tests/incentivization/test_rpc_codec.nim

```diff
@@ -0,0 +1,32 @@
+import
+  std/options,
+  std/strscans,
+  testutils/unittests,
+  chronicles,
+  chronos,
+  libp2p/crypto/crypto
+
+import
+  ../../../waku/incentivization/rpc,
+  ../../../waku/incentivization/rpc_codec
+
```

Comment:
Kindly add this file within the `tests/all_tests_waku.nim` file so that it is considered in our CI tests when a PR is submitted.